.github/workflows/ci-protobuf.yml

```
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # v4.0.0
         with:
           go-version-file: 'go.mod'
-      - uses: arduino/setup-protoc@v1
+      - uses: arduino/setup-protoc@v1 # TSCCR: no entry for repository "arduino/setup-protoc"
```

This needs to be converted to something else per internal discussions.

```suggestion
      - uses: arduino/setup-protoc@v1
```

---

Oh and for clarity, removing the comment with the hopes the automation will raise a separate PR to remind us to update this by itself.